<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>973</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Restaurant Settings</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit">
        <property name="placeholderText">
         <string>Restaurant Name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Save Changes</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Restaurant Name</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_tables">
       <property name="title">
        <string>Tables</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox_addTable">
          <property name="title">
           <string>Add Table</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLineEdit" name="lineEdit_tableName">
             <property name="text">
              <string/>
             </property>
             <property name="placeholderText">
              <string>Table Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_addTable">
             <property name="text">
              <string>Add Table</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_removeTable">
          <property name="title">
           <string>Remove Table</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QListWidget" name="listWidget_tables"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_removeTable">
             <property name="text">
              <string>Remove Table</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_products">
       <property name="title">
        <string>Products</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox_addProduct">
          <property name="title">
           <string>Add Product</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLineEdit" name="lineEdit_productName">
               <property name="placeholderText">
                <string>Product Name</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="doubleSpinBox_productPrice">
               <property name="layoutDirection">
                <enum>Qt::RightToLeft</enum>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="singleStep">
                <double>0.500000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>â‚º</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_addProduct">
             <property name="text">
              <string>Add Product</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_removeProduct">
          <property name="title">
           <string>Remove Product</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QListWidget" name="listWidget_products"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_removeProduct">
             <property name="text">
              <string>Remove Product</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_tableName</tabstop>
  <tabstop>pushButton_addTable</tabstop>
  <tabstop>listWidget_tables</tabstop>
  <tabstop>pushButton_removeTable</tabstop>
  <tabstop>lineEdit_productName</tabstop>
  <tabstop>doubleSpinBox_productPrice</tabstop>
  <tabstop>pushButton_addProduct</tabstop>
  <tabstop>listWidget_products</tabstop>
  <tabstop>pushButton_removeProduct</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
